usr/src/uts/common/fs/zfs/dmu_objset.c

```diff
@@ -67,6 +67,13 @@ krwlock_t os_lock;
  */
 int dmu_find_threads = 0;
 
+/*
+ * Backfill lower metadnode objects after this many have been freed.
+ * Backfilling negatively impacts object creation rates, so only do it
+ * if there are enough holes to fill.
+ */
+int dmu_rescan_dnode_threshold = 1 << DN_MAX_INDBLKSHIFT;
```

Comment:
yes it's just a count and not directly related to DN_MAX_INDBLKSHIFT. I'll set it to 131072 instead.